src/aerovaldb/const.py

```diff
@@ -0,0 +1 @@
+IMG_FILE_EXTS = set([".bmp", ".jpeg", ".jpg", ".png", ".tiff"])
```

Comment:
This is a strange list of image file-types. I don't find `.tiff` or `.bmp` in the current aeroval database, and I don't expect that we will put them there. I'd rather expect modern web-images, in particular svg, webp and avif:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#image_types

What was the origin of this list?

Reply:
I've changed it to follow the web-safe list on that link now.